GiTracker/Services/Rest/RequestSettings.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace GiTracker.Services.Rest
+{
+    public class RequestSettings
```

Comment:
I propose to simplify that thing. Move user-agent to constants, HostName too (yeah, in future we will support more that one server, but lets keep the development step by step). I think that there is no reason to have a class for Url only, so I propose to remove it.